hail/src/main/scala/is/hail/expr/ir/Compile.scala

```diff
@@ -43,6 +43,7 @@ object Compile {
       .zipWithIndex
       .foldLeft(Env.empty[IR]) { case (e, ((n, t, _), i)) => e.bind(n, In(i, t)) }))
     ir = LoweringPipeline.compileLowerer.apply(ctx, ir, optimize).asInstanceOf[IR]
+    ir = LowerArrayToStream(ir) //#LoweringPipeline.streamLowerer.apply(ctx, ir, optimize).asInstanceOf[IR]
```

Comment:
this line needs to go away. LowerArrayToStream should be a pass in the compile lowerer.

Reply:
it will, this part isn't done yet, I think I mentioned the last bit was to add the lowering pass to compileLowerer